diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -371,20 +372,23 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         SECONDARY_CONSTRUCTOR -> parent(CLASS)!!
         else -> treeParent
     }
-        .getIdentifierName()?.text
+            .getIdentifierName()?.text
 
     private fun ASTNode.getOrderedCallExpressions(psi: PsiElement, result: MutableList<ASTNode>) {
         // if statements here have the only right order - don't change it
 
-        if (psi.children.isNotEmpty() && (psi.children[0].node.elementType != DOT_QUALIFIED_EXPRESSION
-                        && psi.children[0].node.elementType != SAFE_ACCESS_EXPRESSION )) {
-            result.add(psi.children[0].node.siblings(true)
+        if (psi.children.isNotEmpty() && (psi.firstChild.node.elementType != DOT_QUALIFIED_EXPRESSION
+                        && psi.firstChild.node.elementType != SAFE_ACCESS_EXPRESSION)) { 
+                            if (psi.firstChild.node.elementType == POSTFIX_EXPRESSION) {
+                                result.add(psi.firstChild.node)
```

Comment:
Doesn't calling `result.add` twice in some cases lead to some sort of data duplication?

Reply:
No, because in the second case it takes another node.